kubetest/gke.go

```diff
@@ -581,12 +613,18 @@ func (g *gkeDeployer) Down() error {
 	if errFirewall != nil {
 		return fmt.Errorf("error deleting firewall: %v", errFirewall)
 	}
+	if errCleanFirewalls != nil {
+		return fmt.Errorf("error cleaning-up firewalls: %v", errCleanFirewalls)
+	}
 	if errSubnet != nil {
 		return fmt.Errorf("error deleting subnetwork: %v", errSubnet)
 	}
 	if errNetwork != nil {
 		return fmt.Errorf("error deleting network: %v", errNetwork)
 	}
+	if numLeakedFWRules > 0 {
+		return fmt.Errorf("leaked firewall rules!")
```

Comment:
Might be better to log which firewall rules are leaked.

Reply:
The leaked firewall rules are already logged from within cleanupNetworkFirewalls method. Should it also be added to error message?

Reply:
Yeah you are right, they are logged already. I scanned too quick.

Reply:
eh, question here, those firewall-rules are actually being cleaned, why we still throw an error here?

cc @yliaog 
ref https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/istio-periodic-e2e-gke-addon/1542/

Reply:
This comes from 2 assumptions:
- you want to be informed when something doesn't work correctly
- you want network (and other resources) to be cleaned, so it will no impact other executions

Even though the network is cleaned, it had to remove leaked resources. I consider resource leakage as a test failure thus propagating it to the error level.

Reply:
ah I see.. I'd suggest we log down the name of leaked firewall rules as well, aka return a list of fw-names here, so we can have better info to debug